VogtPayroll2/PayrollManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    class PayrollManager
+    {
+        private List<Employee> _employees = new List<Employee>();
+
+        public void GetNetPayOfAllEmployees(Employee[] employee)
+        {
+            // The average net pay of all employees should also be displayed
+        }
+
+        public decimal GetMinimumAndMaximumNetPay()
+        {
+            // In addition, the program should find the minimum and maximum net pay of all employees as well as sort the employees based on their net pay (ascending order)
+            // The program must combine two sorting techniques; the Selection and Exchange sorts. It will efficiently sort the employees' net pay.
+            // Preferrably without using a data file.
+            decimal netPay = default;
+
+            return netPay;
+
+        }
+
+        public List<Employee> AddEmployee(Employee employee)
+        {
+            _employees.Add(employee);
```

Comment:
Awesome! There isn't necessarily a reason to return the employee here, as whoever is calling this already had to pass in the employee -- so they'll already have the reference to the employee.

I'd make this:
```c#
public void AddEmployee(Employee employee);
```